include/cudaq/Optimizer/Builder/Factory.h

```diff
@@ -197,6 +197,19 @@ createInvariantLoop(mlir::OpBuilder &builder, mlir::Location loc,
                                             mlir::Region &, mlir::Block &)>
                         bodyBuilder);
 
+/// Builds a monotonic loop. A monotonic loop is a loop that is guaranteed to
+/// execute the body of the loop from \p start to (but not including) \p stop
+/// stepping by \p step times. Exceptional conditions will cause the loop body
+/// to execute 0 times. Early exits are not allowed. This builder threads the
```

Comment:
> Early exits are not allowed.

Does this mean that `break` statements are not permitted?

Reply:
It may behave correctly, but it is highly discouraged.